src/main/java/nl/tudelft/dnainator/ui/DNAinator.java

```diff
@@ -24,20 +26,31 @@
 	private static final int MIN_WIDTH = 300;
 	private static final int MIN_HEIGHT = 150;
 
+	private Stage stage;
+
 	@Override
 	public void start(Stage primaryStage) throws Exception {
-		primaryStage.setMinHeight(MIN_HEIGHT);
-		primaryStage.setMinWidth(MIN_WIDTH);
-		primaryStage.setTitle(DNAINATOR);
-		primaryStage.getIcons().add(new Image(getClass().getResourceAsStream(ICON)));
-		primaryStage.setMaximized(true);
```

Comment:
Right now, you are only setting the window to the screen's dimensions. This is not a "truly maximized" window. You may try setting maximized after setting the window's dimensions.